debian/bitlbee-common.preinst

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+NAME=bitlbee
+PIDFILE=/var/run/$NAME.pid
+
+# git revision 7949d5a introduced dh_systemd.
+DH_SYSTEMD_REV="3.4.2+20160612+master+41-g7949d5a-git"
```

Comment:
>There is no guarantee that the user will be using systemd. How would this work with e.g. sysvinit or upstart?

This particular preinst script doesn't depend on systemd, it uses the pidfile written by bitlbee to kill the master process using sysvinit-style commands. It's just to let systemd start the process normally because the sysvinit scripts didn't stop it.